### DISCUSSION TITLE
瞧瞧问下什么时候支持Hysteria2的端口跳跃。

### DISCUSSION THREAD

**Discussion by klmmlk** [immortalwrt/homeproxy#122] [General]
我原以为是singbox不支持，刚刚查了一下[singbox已支持](https://github.com/fscarmen/sing-box/issues/84#issuecomment-2102786996)，现在联通天天封端口，我真的是要吐了。
求大佬不忙的时候搞一搞。

**Comment by 1715173329** [comment]
https://sing-box.sagernet.org/configuration/outbound/hysteria2/

sing-box 不支持 hy2 端口跳跃，其他应用均为自行操作防火墙实现。
我不使用 hy2，如有需求请自行 PR。
